src/text-expander-element.ts

```diff
@@ -129,6 +129,8 @@ class TextExpander {
 
     this.input.value = beginning + value + remaining
 
+    this.input.dispatchEvent(new Event('change', {bubbles: true}))
```

Comment:
The tests run green for me with this line removed so either the tests are wrong or the implementation.